auth-web/public/index.html

```diff
@@ -4,7 +4,7 @@
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width,initial-scale=1.0">
-    <title>Cooperatives Online</title>
+    <title>Business Registry</title>
```

Comment:
Not "BC Business Registry" ?

Reply:
'Business Registry ' According to the req doc, but i'll follow up!

Reply:
Updated in next commit!